docs/releasing-files.md

```diff
@@ -270,6 +270,9 @@ Only certain file types will be reviewed and released from the secure server. Se
 
 If you would like to release other file types, please email <datarelease@opensafely.org>, stating why it is important that the file is released in a different format.
 
+!!! note
+    The maximum file size that can be released is 16mb. Please check your outputs before requesting them for release. It is unlikely any outputs that exceed this in size are appropriate for release, but if you think they are, please let us know when making a release request.
```

Comment:
Could write as 16 MB to be consistent with this section of the documentation https://docs.opensafely.org/actions-scripts/#file-size-restrictions, but also fine as-is